build.proj

```diff
@@ -80,8 +80,9 @@
 
   <Import Project="build/Microsoft.DotNet.Cli.Prepare.targets" />
   <Import Project="build/Microsoft.DotNet.Cli.Compile.targets" />
+  <Import Project="build/Microsoft.DotNet.Cli.Signing.targets" />
   <Import Project="build/Microsoft.DotNet.Cli.Package.targets" />
-  <Import Project="build/Microsoft.DotNet.Cli.Test.targets" />
+  <Import Project="build/Microsoft.DotNet.Cli.Test.targets" Condition="'$(SkipMsBuild15Projects)' != 'True'" />
```

Comment:
I can look into the possibility of getting MSBuild 15 on the build agents and then remove this code at that point, but I wanted to get this working now.